### PR TITLE
Exclude migrations from Rails/SkipModelValidations.

### PR DESCRIPTION
**Pull request by drn**

### CODE REVIEW

File: rails/rubocop.yml [hunk 1]
```diff
@@ -9,3 +9,7 @@ Rails/UnknownEnv:
     - production
     - staging
     - demo
+
+Rails/SkipsModelValidations:
```
Comment (marcferna) [inline rails/rubocop.yml]: i've been keeping this in alphabetical oder

**Review by marcferna — APPROVED** [review]
Sort on alphabetical order and Ship it

File: rails/rubocop.yml (hunk 1 above)
Comment (drn) [reply rails/rubocop.yml]: 👍 can we lint our linter config!?! 😉

Comment (marcferna) [reply rails/rubocop.yml]: I can't find one but would be good!

Comment (drn) [reply rails/rubocop.yml]: LOL lintception
if we really wanted to, we can add a quick ruby script (or w/e language) that loads in all the `rubcop.yml` files via `YAML.load_file` and runs through all the keys...

**Pull request merged**